Tests/AsyncHTTPClientTests/HTTPClientInternalTests.swift

```diff
@@ -424,11 +424,10 @@ class HTTPClientInternalTests: XCTestCase {
         let randoEL = group.next()
 
         let httpClient = HTTPClient(eventLoopGroupProvider: .shared(group))
-        let promise: EventLoopPromise<Channel> = httpClient.eventLoopGroup.next().makePromise()
-        let httpBin = HTTPBin(channelPromise: promise)
+        let server = NIOHTTP1TestServer(group: MultiThreadedEventLoopGroup(numberOfThreads: 1))
```

Comment:
aren't we leaking the `group` here? It needs to be stopped.

Reply:
oops, we do

Reply:
fixed